portfolio/src/main/webapp/projects/musicplayer/ipod.js

```diff
@@ -0,0 +1,113 @@
+// Create your global variables below:
+var tracklist = ["All We Got","No Problem", "Summer Friends", "D.R.A.M. Sings Special", "Blessings", "Same Drugs", "Mixtape", "Angels", "Juke Jam", "All Night"];
+var volLevels = []; 
+var colored; //variable to keep track of what the volume is currently filled in
+var timeVar = null; //initializes the setInterval variable
+var currentsong; //varibale to keep track of current song playing
+
+
+function init() {
+	volumelevel='vl';
+	for(var i=0;element= document.getElementById(volumelevel+i);i++){
```

Comment:
using `element = ` without a `var` or `let` anywhere I believe means that `element` is assigned globally here.

Using an assignment in the for-loop condition like this is confusing and kind of weird. If you want to do an assignment in the loop condition, I would make it explicit by wrapping the assignment in parenthesis and using an explicit `!= null` or `!!(element = ...)` to make it clear that you are converting to boolean. (`!!` is the boolean-negation operator twice, which is used in JavaScript to cast other types to boolean explicitly).